csharp/README.md

```diff
@@ -1,206 +1,45 @@
-# Building Ice for .NET
+# Ice for .NET
 
-This page describes how to build Ice for .NET from source and package the
-resulting binaries. As an alternative, you can download and install the
-[zeroc.ice.net][1] NuGet package.
+[Getting started] | [Examples] | [NuGet packages] | [Documentation] | [Building from source]
 
-* [Building on Windows](#building-on-windows)
-  * [Windows Build Requirements](#windows-build-requirements)
-  * [Compiling Ice for \.NET on Windows](#compiling-ice-for-net-on-windows)
-    * [Strong Name Signatures for \.NET Framework 4\.5 Assemblies](#strong-name-signatures-for-net-framework-45-assemblies)
-    * [Authenticode Signatures](#authenticode-signatures)
-    * [Building only the Test Suite](#building-only-the-test-suite)
-* [Building on Linux or macOS](#building-on-linux-or-macos)
-  * [Linux and macOS Build Requirements](#linux-and-macos-build-requirements)
-  * [Compiling Ice for \.NET on Linux or macOS](#compiling-ice-for-net-on-linux-or-macos)
-* [Running the Tests](#running-the-tests)
-* [NuGet Package](#nuget-package)
+Ice is a full featured framework for creating networked applications with RPC, pub/sub, server deployment
+and more.
 
-## Building on Windows
+## Sample Code
 
-A source build of Ice for .NET on Windows produces two sets of assemblies:
- - assemblies for the .NET Framework 4.5
- - assemblies for [.NET Standard 2.0][2]
-
-### Windows Build Requirements
-
-In order to build Ice for .NET from source, you need:
- - A [supported version][3] of Visual Studio when building .NET Framework 4.5 Assemblies.
- - Visual Studio 2022 with [.NET 6.0 SDK][4] or [.NET 7.0 SDK][5] when building the .NET Standard 2.0 Assemblies.
- - Visual Studio 2022 with [.NET 6.0 SDK][4] and [.NET 7.0 SDK][5] when building the NuGet packages.
-
-### Compiling Ice for .NET on Windows
-
-Open a Visual Studio command prompt and change to the `csharp` subdirectory:
-```
-cd csharp
-```
-
-To build all Ice assemblies and the associated test suite, run:
-```
-msbuild msbuild\ice.proj
-```
-
-Upon completion, the Ice assemblies for the .NET Framework 4.5 and .NET Standard 2.0 are placed
-in the `lib\net45` and `lib\netstandard2.0` folders respectively.
-
-You can skip the build of the test suite with the `BuildDist` target:
-```
-msbuild msbuild\ice.proj /t:BuildDist
+```slice
+module Demo
+{
+    interface Hello
+    {
+        idempotent void sayHello(int delay);
+        void shutdown();
```

Comment:
You should use the minimal demo interface: no idempotent, no delay, no shutdown.